```
Traceback (most recent call last):
  File "/mnt/disk1/wangyi/transformers/test.py", line 23, in <module>
    main()
  File "/mnt/disk1/wangyi/transformers/test.py", line 17, in main
    speech = generator("Hello, my dog is cute", forward_params={"tgt_lang": "rus"})
  File "/mnt/disk1/wangyi/transformers/src/transformers/pipelines/text_to_audio.py", line 182, in __call__
    return super().__call__(text_inputs, **forward_params)
  File "/mnt/disk1/wangyi/transformers/src/transformers/pipelines/base.py", line 1206, in __call__
    return self.run_single(inputs, preprocess_params, forward_params, postprocess_params)
  File "/mnt/disk1/wangyi/transformers/src/transformers/pipelines/base.py", line 1214, in run_single
    outputs = self.postprocess(model_outputs, **postprocess_params)
  File "/mnt/disk1/wangyi/transformers/src/transformers/pipelines/text_to_audio.py", line 204, in postprocess
    output_dict["audio"] = waveform.cpu().float().numpy()
AttributeError: 'tuple' object has no attribute 'cpu
```

--- a/src/transformers/pipelines/text_to_audio.py
+++ b/src/transformers/pipelines/text_to_audio.py
@@ -200,7 +200,8 @@ class TextToAudioPipeline(Pipeline):
 
     def postprocess(self, waveform):
         output_dict = {}
-
+        if isinstance(waveform, tuple):
+            waveform = waveform[0]
         output_dict["audio"] = waveform.cpu().float().numpy()
         output_dict["sampling_rate"] = self.sampling_rate
 

